spec/models/factory_girl_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe 'FactoryGirl' do
+  local_models = Dir.glob('app/models/*.rb').map { |file| File.basename(file, '.rb') }
+
+  ApplicationRecord.subclasses.each do |klass|
```

Comment:
Should this be `ApplicationRecord.descendants`?  `subclasses` only gets you the first level of the hierarchy.

Reply:
☝️